flutter/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -42,5 +42,7 @@
         <meta-data
             android:name="flutterEmbedding"
             android:value="2" />
+
+        <meta-data android:name="io.sentry.traces.profiling.enable" android:value="true" />
```

Comment:
Workaround for https://github.com/getsentry/sentry-dart/pull/851/files#r860567670